File: 30-Days-of-Code/Day-2-Operators/Fortran/Solution.f
real :: mealCost
integer :: tipPercent, taxPercent, roundedTotalCost

READ *, mealCost
READ *, tipPercent
READ *, taxPercent

tip = (tipPercent / 100.0) * mealCost
tax = (taxPercent / 100.0) * mealCost

totalCost = mealCost + tip + tax

roundedTotalCost = INT(NINT(totalCost))

PRINT 100, roundedTotalCost
100 FORMAT ("The total meal cost is ", I2, 1x, "dollars.")

END

File: 30-Days-of-Code/Day-2-Operators/Erlang/solution.erl
-module(solution).
-export([main/0]).

main() ->
    {MealCost, _} = string:to_float(string:strip(io:get_line(""), both, $\n)),
    {TipPercent, _} = string:to_integer(string:strip(io:get_line(""), both, $\n)),
    {TaxPercent, _} = string:to_integer(string:strip(io:get_line(""), both, $\n)),  

    Tip = MealCost * (TipPercent / 100),
    Tax = MealCost * (TaxPercent / 100),
    
    TotalCost = MealCost + Tip + Tax, 
    
    RoundedTotalCost = round(TotalCost),
    
    io:format("The total meal cost is ~p dollars.\n", [RoundedTotalCost]).
